src/schemathesis/cli/__init__.py

```diff
@@ -235,16 +237,20 @@ class OutputStyle(Enum):
 
     default = output.default.DefaultOutputStyleHandler
     short = output.short.ShortOutputStyleHandler
+    junit_xml = output.junitxml.JunitXMLHandler
```

Comment:
It shouldn't be here, these are only for output handlers. JUnit should go to a file